quetz/migrations/versions/ea6eba9a9ffc_merge_ebe550f9fbbe_and_b9886d9cadb0.py

```diff
@@ -0,0 +1,21 @@
+"""merge ebe550f9fbbe and b9886d9cadb0
+
```

Comment:
This one is completely empty?

Reply:
yeah it just merges 2 migrations branching from the same root.

Reply:
This is analogous to a merge commit. And is the standard way according to @btel 
Alternatively I manually edited it to preserve a linear history but that is dangerous if the branched migrations are deployed on beta.mamba.pm

Reply:
all good, thanks for the explanation!